docker/README.md

```diff
@@ -0,0 +1,32 @@
+# ICLR Blogpost 2022 Dockerfile
+ 
+This repository defines a docker image that allows users to serve and build Jekyll/Github-Pages sites
+for the ICLR blog track.
+The image can be found on dockerhub:
+
+- [velythyl/jekyll-ghp](https://hub.docker.com/r/velythyl/jekyll-ghp)
+
+Of particular note is `ghp_export.sh`. This implements our specific logic for hosting the submitters static site:
+we generate a unique ID, concatenate a timestamp to it, and then replace the Jekyll URL inside the config folder by it.
+
+# Build
```

Comment:
I would remove line 12 and onward. Telling them to build just introduces uncertainty (they might erroneously overwrite the velyhtyl/jekyll-ghp image on their local computer). Push is also completely useless to them.

I'd replace it by 'you can use the docker image by calling the commands defined in the makefile'